### PR TITLE
feat(ncp): add `ncp` which uses `ncp`

### LINKED ISSUE #20: cpy-cli fails to copy a folder recursively
Can't seem to copy folders with subfolders recursively using the built-in [cpy-cli](https://github.com/sindresorhus/cpy-cli). See https://github.com/sindresorhus/cpy-cli/issues/10.

Project is running on Windows.

### PR DESCRIPTION
**Pull request by indfnzo**
Adds an alternative util to `copy (cpy-cli)` using `ncp` to allow recursive copying of directories.

closes #20

### CODE REVIEW

**Comment by indfnzo** [comment]
Seems like I screwed something up with the testing environment, HAHA.
Should I have not modified the snapshot?

**Comment by kentcdodds** [comment]
Hmmm... It looks like you're on a windows machine. We need to normalize the paths for our snapshots. I think we could do that in [this function](https://github.com/kentcdodds/nps-utils/blob/6753ad57e2f636a79b25d1ca6b942cadc9f49462/src/index.test.js#L80-L85). Just want to make sure the slashes in the path are the same across platforms. I'm not totally sure of the best way to do that. Would you like to look into it? Thanks!

**Comment by indfnzo** [comment]
Sure, I just have some things to settle right now. I'll try to get to this some time this week.

File: src/index.test.js [hunk 1]
```diff
@@ -78,10 +79,20 @@ function withPlatform(platform, getResult) {
 }
 
 function relativeizePath(stringWithAbsolutePaths) {
-  return stringWithAbsolutePaths.replace(
-    new RegExp(path.resolve(__dirname, '../'), 'g'),
+  // escape string for regexp generation
+  let escapedPath = path.resolve(__dirname, '../').replace(/[\-\[\]\/\{\}\(\)\*\+\?\.\\\^\$\|]/g, "\\$&");
```
Comment (kentcdodds) [inline src/index.test.js]: 🤷‍♂️ seems to work alright

**Review by kentcdodds — APPROVED** [review]
Looks good to me. Thanks!

**Comment by kentcdodds** [comment]
Hmm... The build is breaking due to eslint errors. Could give those a look? Thanks!

**Comment by indfnzo** [comment]
Just now I realized I could lint locally with `nps lint`, HAHA. Anyway, everything seems in place.

**Review by kentcdodds — APPROVED** [review]
Thank you!

**Comment by indfnzo** [comment]
:)

**Comment by kentcdodds** [comment]
Thanks so much for your help! I've added you as a collaborator on the project. Please make sure that you review the `other/MAINTAINING.md` and `CONTRIBUTING.md` files (specifically the bit about the commit messages and the git hooks) and familiarize yourself with the code of conduct (we're using [the contributor covenant](http://contributor-covenant.org/)). You might also want to [watch the repo](https://help.github.com/articles/watching-repositories/) to be notified when someone files an issue/PR. Please continue to make PRs as you feel the need (you can make your branches directly on the repo rather than your fork if you want). Thanks! And welcome to the team :)

**Comment by indfnzo** [comment]
Great, thanks!

I missed the part about adding myself as a contributor manually, so it's actually not part of this pull request. Other than that, I believe I was able to follow all the rules.

**Comment by kentcdodds** [comment]
Feel free to open another pull request to add yourself 👍

**Pull request merged**